content/momentum/changelogs/M4ChangeLog.md

```diff
@@ -19,6 +21,219 @@
   <tr><td><a href="#4.0.0">Momentum 4.0.0 released on 2014-04-30</a></td></tr>
 </table>
 
+<h2><a name="4.4.0">Momentum 4.4.0 released on 2021-08-?? (Depending on installation type, all changes may not be applicable)</a></h2>
+<ul>
```

Comment:
Some more:

fix: SD-2606 spf_parser_v1 can malloc size 0
fix: SD-2574 resident message count may be incorrect
fix: SD-2350 messages may be incorrectly rejected with relaying denied
Feature: SD-2743 introduced msg_gen scheduled send functionality

Reply:
Feature: SD-2365 introduced HA Proxy client module

Reply:
Finding more looking at the tickets in the 4.4. epic:

SD-2458 Added support for openssl fips_mode
SD-2563 Introduced tls_params lua module
SD-2744 Introduced xfer delivery module
SD-2712 Config Option to disable the bind_addresses IP check
SD-2634 outbound_smtp_auth fails when sending more than one msg per connection
SD-1170 Upgraded gnutls to version 3.3.0 and nghttp2 to 1.44.0